### PR TITLE
[chore] add issue templates

### PR DESCRIPTION
**Pull request by rougetimelord**
maybe this will help people write better issues?

### CODE REVIEW

**Comment by cooljeanius** [comment]
Maybe include a reference to #349 in the bug report one?

**Comment by rougetimelord** [comment]
> Maybe include a reference to #349 in the bug report one?

Not directly mentioning the issue, but I think that's good for flexibility. Hopefully I didn't word it too passive aggressively :P

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 1]
```diff
@@ -0,0 +1,43 @@
...
+name: Bug report
+about: Create a report to help us improve
+title: "[bug]"
+labels: bug
+assignees: ''
+
+body:
+- type: markdown
+  attributes:
+    value: "Hi! Thank you for your bug report, for our own sanity we want to ask you a question."
```
Comment (cooljeanius) [inline .github/ISSUE_TEMPLATE/bug_report.md:11]:
```suggestion
    value: "Hi! Thank you for your bug report. We have a question for you to answer first."
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 2]
```diff
@@ -0,0 +1,43 @@
+---
+name: Bug report
+about: Create a report to help us improve
```
Comment (cooljeanius) [inline .github/ISSUE_TEMPLATE/bug_report.md:3]:
```suggestion
about: Create a report to help us improve Blue-Blocker
```

File: .github/ISSUE_TEMPLATE/bug_report.md [hunk 3]
```diff
@@ -0,0 +1,43 @@
...
+1. Go to '...'
+2. Click on '....'
+3. Scroll down to '....'
+4. See error
+
+**Expected behavior**
+A clear and concise description of what you expected to happen.
+
+**Screenshots and console logs**
+If applicable, add screenshots to help explain your problem. If there's console logs related to the bug  add them here too.
```
Comment (cooljeanius) [inline .github/ISSUE_TEMPLATE/bug_report.md:36]:
```suggestion
If applicable, add screenshots to help explain your problem. If there are console logs related to the bug, then please add them here, too.
```

**Review by cooljeanius — CHANGES_REQUESTED** [review]

**Review by cooljeanius — APPROVED** [review]

**Review by cooljeanius — APPROVED** [review]

**Pull request merged**